### ISSUE TITLE
How does Save Preset work?

### ISSUE THREAD

**Issue by cellio** [mattjbrent/mattjbrent.github.io#5]
What is supposed to happen when I click "Save Preset"?  I'm imagining that I'd get prompted for a name to use, and then if I click on "Select Preset" I'll get a menu of all the ones I've defined (and maybe some baked-in ones?).  Is that right?  We don't need to implement it for the mockup, but it'd be good to explain how it's intended to work in that case.

**Comment by mattjbrent** [comment]
Yes exactly that @cellio. I was thinking a modal to create your preset and then just a simple dropdown to select a preset.

I think there'll need to be more work done on the filter preset. i dont see it too useful to net out now though.
